### PR TITLE
Oracle GoldenGate in a Container

### PR DESCRIPTION
**Pull request by sbalousek**
- Allow GoldenGate images to be created with an alternate base image
 - Minor corrections to documentation

Signed-off-by: Stephen Balousek <stephen.balousek@oracle.com>

### CODE REVIEW

**Review by Djelibeybi — APPROVED** [review]

**Comment by Djelibeybi** [comment]
Could you please add an ignore directive so that the linter check passes?

**Comment by sbalousek** [comment]
The linter check is passing now. Is there Anything else you would like to see here, @Djelibeybi ?

**Comment by Djelibeybi** [comment]
Nope, but until you push the "Re-request review" button (which I just did), I don't get notified that it has been updated. :) Reviewing now.

File: OracleGoldenGate/21c/README.md [hunk 1]
```diff
@@ -41,18 +44,30 @@ To create a container image for GoldenGate for Oracle Database, use the followin
...
 ...
 Successfully tagged oracle/goldengate:21.3.0.0.0
 ```
 Similarly, for other Databases like BigData, MySQL, PostgreSQL, etc. provide the name of the zip file for the INSTALLER argument.
 
+### Changing the Base Image
+
+By default, the base image used by Docker to build Oracle GoldenGate Docker image is `oraclelinux:8`, but this can be changed.
+
+Changing the base image for the new Docker image is done with the `BUILD_IMAGE` Docker build argument, like in this example:
```
Comment (Djelibeybi) [inline OracleGoldenGate/21c/README.md:58]: Please remove "Docker" from this sentence, e.g. "Changing the base image for the new container image is done using the `BUILD_IMAGE` argument when building the image. For example:"

File: OracleGoldenGate/21c/README.md [hunk 2]
```diff
@@ -41,18 +44,30 @@ To create a container image for GoldenGate for Oracle Database, use the followin
...
+               --build-arg INSTALLER=213000_fbo_ggs_Linux_x64_Oracle_services_shiphome.zip .
 Sending build context to Docker daemon
 ...
 Successfully tagged oracle/goldengate:21.3.0.0.0
 ```
 Similarly, for other Databases like BigData, MySQL, PostgreSQL, etc. provide the name of the zip file for the INSTALLER argument.
 
+### Changing the Base Image
+
+By default, the base image used by Docker to build Oracle GoldenGate Docker image is `oraclelinux:8`, but this can be changed.
```
Comment (Djelibeybi) [inline OracleGoldenGate/21c/README.md:56]: Please remove "Docker" from this sentence, i.e. "By default, the base image used by the Oracle GoldenGate container image is `oraclelinux:8` but this can be changed."

It would also be good to provide either a list of valid base images or criteria by which someone could determine if something would work as a base image.

File: OracleGoldenGate/21c/README.md [hunk 3]
```diff
@@ -143,13 +158,15 @@ Oracle GoldenGate Administration Client for Oracle
 Version 21.3.0.0.0 ...
 ```
 
-### Running scripts before setup and on startup
-The container images can be configured to run scripts before setup and on startup. Currently, `.sh` extensions are supported. For setup scripts just mount the volume `/u01/ogg/scripts/setup` or extend the image to include scripts in this directory. For startup scripts just mount the volume `/u01/ogg/scripts/startup` or extend the image to include scripts in this directory. Both of those locations are static and the content is controlled by the volume mount.
+### Running Scripts Before Setup and on Startup
+
+The container images can be configured to run scripts before setup and on startup. Currently, `.sh` extensions are supported. For setup scripts just mount the volume `/u01/ogg/scripts/setup` or extend the image to include scripts in this directory. For startup scripts just mount the volume `/u01/ogg/scripts/startup` or extend the image to include scripts in this directory. Both of those locations
```
Comment (Djelibeybi) [inline OracleGoldenGate/21c/README.md:163]: I would rephrase this as, "Containers created from this image are configured to automatically run scripts found at specific locations at particular points during initial container creation and on subsequent startups."

Then you'd have two blocks: "To run a script before the initial setup of Oracle GoldenGate ......" and  "To run a script each time the container starts but before Oracle GoldenGate is started...."

File: OracleGoldenGate/21c/README.md [hunk 4]
```diff
@@ -143,13 +158,15 @@ Oracle GoldenGate Administration Client for Oracle
...
+are static and the content is controlled by the volume mount.
 
 The example below mounts the local directory myScripts to `/u01/ogg/scripts` which is then searched for custom startup scripts:
```
Comment (Djelibeybi) [inline OracleGoldenGate/21c/README.md:166]: myScripts needs to be wrapped in back ticks and is missing a leading forward slash.

**Review by Djelibeybi — CHANGES_REQUESTED** [review]

File: OracleGoldenGate/21c/README.md (hunk 3 above)
Comment (sbalousek) [reply OracleGoldenGate/21c/README.md:163]: I agree the language here is not the best, but the changes in this PR are limited to:

1. Fix capitalization on the section title
2. Shorten lines to make super-linter happy.

**Review by Djelibeybi — APPROVED** [review]
